v2/community/reusableMD/supported-tech-stacks-backend.mdx

```diff
@@ -5,7 +5,7 @@
   justifyContent: "start",
   flexWrap: "wrap"
 }}>
-<img src="/img/logos/node.svg" style={{width: "100px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/python.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/go.svg" style={{width: "100px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
+<img src="/img/logos/node.svg" alt="nodejs logo" style={{width: "100px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```

Comment:
```suggestion
<img src="/img/logos/node.svg" alt="Node.js logo" style={{width: "100px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```

Should ideally use the case used on their official website for all the logos

Reply:
done